python-sdk/pachyderm_sdk/client.py

```diff
@@ -85,17 +85,14 @@ def __init__(
             auth_token = os.environ.get(AUTH_TOKEN_ENV)
 
         tls = tls or (root_certs is not None)
-        if tls and root_certs is None:
```

Comment:
Can't comment on the line, but you'll want to update the class docstring to remove the reference to certifi